spec/requests/api/physical_servers_spec.rb

```diff
@@ -0,0 +1,160 @@
+RSpec.describe "physical_servers API" do
+  describe "display a physical server's details" do
+    context "with valid properties" do
+      it "shows all of its properties" do
+        FactoryGirl.create(:physical_server, :id => 1, :ems_ref => "A59D5B36821111E1A9F5E41F13ED4F6A")
+
+        api_basic_authorize action_identifier(:physical_servers, :read, :resource_actions, :get)
+
+        run_get "/api/physical_servers/1"
```

Comment:
run_get physical_servers_url(1)  

you can also use the id generated by above FactoryGirl (without :id => 1), i.e. ps = FactoryGirl.create(...), then use physical_servers_url(ps.id)